packages/cfd/src/Containers/__tests__/compare-accounts-content.spec.js

```diff
@@ -0,0 +1,206 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import CompareAccountsContent from '../compare-accounts-content';
+
+describe('should render the CompareAccountsContent component properly', () => {
+    let modal_root_el;
+
+    beforeAll(() => {
+        modal_root_el = document.createElement('div');
+        modal_root_el.setAttribute('id', 'modal_root');
+        document.body.appendChild(modal_root_el);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(modal_root_el);
+    });
+
+    const all_landing_companies = {
+        mt_gaming_company: {
+            financial: 'USD',
+        },
+        mt_financial_company: {
+            financial: 'USD',
+            financial_stp: 'USD',
+        },
+    };
+
+    let mock_props;
+
+    beforeEach(() => {
+        mock_props = {
+            is_logged_in: true,
+            platform: 'mt5',
+            show_eu_related: 'false',
+            residence: 'idn',
+            is_eu: 'false',
+            is_uk: 'false',
+        };
+    });
+
+    it('should render the component', () => {
+        render(<CompareAccountsContent {...mock_props} landing_companies={all_landing_companies} />);
+    });
```

Comment:
There is no assertion for this test case, so the test will pass even if component is not rendered